lib/iex/lib/iex/introspection.ex

```diff
@@ -735,9 +735,12 @@ defmodule IEx.Introspection do
     string = "@#{kind} #{Macro.to_string(definition)}"
 
     try do
+      whitespace = String.duplicate(" ", nesting)
+
       string
       |> Code.format_string!(line_length: IEx.width() - 2 * nesting)
       |> IO.iodata_to_binary()
+      |> String.replace("\n", "\n#{whitespace}")
```

Comment:
I am afraid this is going to be indented twice, because we already indent after the try/catch block.